atm.js

```diff
@@ -1,11 +1,105 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//var checkingBalance = document.querySelector("div#checking_balance");
+//checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//var checkingDeposit = function(amount){
+  //checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );}
 
+//add button functions
+  var cDeposit;
+  var checkingWithdrawal;
+  var sDeposit;
+  var savingsWithdrawal;
+  var checkingBalance= 0;
+  var savingsBalance= 0;
+  var totalBalance = checkingBalance + savingsBalance;
+
+
+if (checkingBalance === 0){
+  console.log("zero")
+  $(".balance").eq(0).addClass("zero")
+}
+if (savingsBalance === 0){
+  console.log("zero")
+  $(".balance").eq(1).addClass("zero")
+}
+
+//DEPOSIT IN CHECKING
+$(":button").eq(0).on("click", function checkingDeposit(){
```

Comment:
Probably better to use ids on the buttons to avoid using `.eq(0)` etc; this way, your code is more readable!